website/pages/blog/product-updates-1.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "CloudQuery Product Updates #1"
+tag: product
+date: 2022/11/01
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as figured it easier to use number then months :)
+
+This months we've been really cranking it at CQ so lot's of good stuff!
+
+## CLI Imprvoments
+
+* We've introduced glob to specify tables. Check out release [blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)
```

Comment:
`globbing` instead of "glob"? Though it's not really glob, is it? Maybe:

```suggestion
* We've added wildcard support to table matching. Check out release [blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)
```

table matching. or filtering?